internal/controlplane/handlers_ruletype_test.go

```diff
@@ -180,3 +183,46 @@ func TestCreateRuleType(t *testing.T) {
 		})
 	}
 }
+
+// TODO: replace with gomock
```

Comment:
Can we move this to `github.com/stacklok/minder/internal/events/testrecorder` or the like?  I banged it in somewhere because I needed it once, but it seems fairly simple / reusable.